applications/luci-app-isc-dhcp/root/usr/share/luci/menu.d/luci-app-isc-dhcp.json

```diff
@@ -0,0 +1,28 @@
+{
+	"admin/network/isc-dhcp": {
+		"title": "ISC DHCP",
+		"order": 10,
+		"action": {
+			"type": "alias",
```

Comment:
You could consider using `firstchild` as target type. That will automatically select the first suitable sub-entry with the lowest `order` value.

Reply:
updated.